CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 ## [0.2.2] - 2023-03-31
 
+## Fixed
```

Comment:
this should not be under 0.2.2 release section, can you move it up. or run make changelog to autopopulate this.

Reply:
No need to update changelog in PRs. Changelog can be automatically generated later when we run `make changelog` in main branch.

Reply:
Just following the [commit message guidelines](https://github.com/paralus/paralus/blob/main/CONTRIBUTING.md#commit-message-guidelines) is enough.

Reply:
Alright, I will just remove changes from changelog then